addons/udes_stock/models/procurement.py

```diff
@@ -8,3 +8,42 @@ class ProcurementGroup(models.Model):
 
     # Allow groups to be created
     MSM_CREATE = True
+
+    def _get_orderpoint_domain(self, company_id=False):
+        """ Override order point domain to filter by active_ids
+        """
+        domain = super()._get_orderpoint_domain(company_id=company_id)
+        orderpoints = self.env.context.get('active_ids', None)
+        if orderpoints:
+            domain += [('id', 'in', orderpoints)]
+        return domain
+
+        
+class StockWarehouseOrderPoint(models.Model):
+    _inherit = 'stock.warehouse.orderpoint'
+
+    @api.model
+    def check_order_points(self, use_new_cursor=False, company_id=False):
+        """  Copy of run_scheduler to only check order points """
+        OrderPoint = self.env["stock.warehouse.orderpoint"]
+        ProcurementGroup = self.env["procurement.group"]
+        try:
+            if use_new_cursor:
+                cr = registry(self._cr.dbname).cursor()
+                domain = ProcurementGroup._get_orderpoint_domain()
+                self = OrderPoint.with_context(prefetch_fields=False).search(domain)
+                self = self.with_env(self.env(cr=cr))
+
+                self.sudo()._procure_orderpoint_confirm(
+                    use_new_cursor=use_new_cursor,
+                    company_id=company_id)
+            if use_new_cursor:
```

Comment:
Redundant `if`?

Reply:
me and caleb noticed this too, it exists in 11 too and can probably be removed, i can't see any reason for it existing

Reply:
Looking at the original, I think the `sudo` call above should be dedented, so it is always executed.  But it's the same in 11 so it could be intentional.  Maybe @ws-mps can remember?

Reply:
I agree, seems that is to be dedented, probably a typo we never noticed.